gitlab-runner-17.2.yaml

```diff
@@ -45,27 +45,36 @@ subpackages:
           output: gitlab-runner-helper
           ldflags: -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
-  - name: "gitlab-runner-compat-${{vars.short-package-version}}"
-    description: "Gitlab-runner Compat package for alpine"
+  - name: "gitlab-runner-oci-entrypoint-${{vars.short-package-version}}"
+    description: "Gitlab-runner oci entrypoint for alpine"
     dependencies:
       provides:
-        - gitlab-runner-compat=${{package.full-version}}
+        - gitlab-runner-oci-entrypoint=${{package.full-version}}
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"
           cp dockerfiles/runner/alpine/entrypoint "${{targets.subpkgdir}}"/entrypoint
           chmod 755 "${{targets.subpkgdir}}"/entrypoint
 
+  - name: "gitlab-runner-helper-oci-entrypoint-${{vars.short-package-version}}"
+    description: "Gitlab-runner-helper oci entrypoint for alpine"
```

Comment:
Same as above - not sure why we're mentioning another distro here - can we update this description